strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/services/impl/ConfigurationManagementServiceImpl.java

```diff
@@ -618,4 +624,61 @@ private void modifyInLock(final Consumer<MutableConfiguration> operation,
         }
     }
 
+    @Override
+    public boolean setMaxUploadFileSize(String maxUploadSizeString) throws SizeFormatValidationException
+    {
+        if (!(this.multipartConfigElement instanceof UpdatableMultipartConfigElement))
```

Comment:
I don't think you really need the extra set of brackets here.

Reply:
Operator '!' cannot be applied to 'javax.servlet.MultipartConfigElement'

Change to '!(this.multipartConfigElement instanceof UpdatableMultipartConfigElement)'